docs/index.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
what is this for?

Reply:
The github actions workflow will populate this file with the packaged version of the chart. When adding the repo, helm fetches from this file.